provider/marathon/marathon.go

```diff
@@ -420,6 +447,187 @@ func (p *Provider) hasCircuitBreakerLabels(application marathon.Application) boo
 	return ok
 }
 
+// Map of services properties
+// we can get it with label[serviceName][propertyName] and we got the propertyValue
+type labelServiceProperties map[string]map[string]string
```

Comment:
What name would you like to use instead?  (`labelServiceProperties` is already defined as a `type`, so I'm assuming you don't like the name)